src/main/java/org/prebid/server/bidder/criteo/CriteoSlotIdGenerator.java

```diff
@@ -0,0 +1,14 @@
+package org.prebid.server.bidder.criteo;
+
+import org.springframework.stereotype.Component;
+
+import java.util.UUID;
+
+@Component
+public class CriteoSlotIdGenerator {
```

Comment:
Made this as a separate component to be able to mock id generation and assert returned values in tests

Reply:
1. Usually we don't use spring annotations along the project but rather inject dependencies in configuration.
2. Seems you can use common component for UUID generation - `org.prebid.server.identity.UUIDIdGenerator`.